### PR TITLE
Updates puma server and database config

### PR DESCRIPTION
**Pull request by b-enji-cmd**
### Add
* `N/A`
### Update / Remove
* `config/database.yml` 
* `config/puma.rb`
### Why?
*  `config/database.yml` : Updates production server to reflect AWS EC2 configuration
* `config/puma.rb` : Preloads application on build

Closes # `N/A`

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -75,11 +75,11 @@ test:
...
 # You can use this database configuration with:
 #
-#   production:
-#     url: <%= ENV['DATABASE_URL'] %>
-#
-production:
-  <<: *default
-  database: mesh-backend_production
-  username: mesh-backend
-  password: <%= ENV['MESH-BACKEND_DATABASE_PASSWORD'] %>
```
Comment (jequeza) [inline config/database.yml:85]: Is this password for the whole repo or we each need to obtain our own?

File: config/puma.rb [hunk 2]
```diff
@@ -24,14 +24,14 @@
 # Workers do not work on JRuby or Windows (both of which do not support
 # processes).
 #
-# workers ENV.fetch("WEB_CONCURRENCY") { 2 }
+workers ENV.fetch("WEB_CONCURRENCY") { 2 }
```
Comment (jequeza) [inline config/puma.rb:27]: workers!!! FTW! 🍄

**Review by jequeza — COMMENTED** [review]
Great Job Dev! AWS coming right up!

**Review by Gvieve — APPROVED** [review]
Overall looks great! Thanks so much for walking us through it and exploring this

**Pull request merged**